flow/project.py

```diff
@@ -2266,7 +2533,7 @@ def labels(self, job):
             elif bool(label_value) is True:
                 yield label_name
 
-    def add_operation(self, name, cmd, pre=None, post=None, **kwargs):
+    def add_operation(self, name, cmd, pre=None, post=None):
```

Comment:
I don't think we can just remove the `kwargs` here, since this a popular high-level function. I understand why you needed to remove it during prototyping, but I think it would be very important to find a way to make this work. Otherwise we have to go through a deprecation here.